aguskoll/target-app
# Feature/sign in

### What's in

The sign in with the backend and in the app.
Error messages when the sign in fails.
Unit testing
View binding

### How it looks

https://drive.google.com/file/d/1ZCzOPVzA2flNn5IJbpeHvJBQWduRECZN/view?usp=sharing

### How to test

1. Run this branch
2. Create an account 
3. Go to the sign in page and fill up your data
4. Verify that you can sign in correctly.
5. Sign out and fill your data incorrectly
6. Verify the error message
